### PR TITLE
[picojson] bump to master + update recipe

### PR DESCRIPTION
**Pull request by prince-chrismc**
There's a security fix that I would like... Almost 6 years since the last release

Specify library name and version:  **picojson/cci.20210117**

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by Croydon** [comment]
Did you ask for a new release first?

**Review by mathbunnyru — APPROVED** [review]
I do agree that we should at least ask for a release.

**Comment by prince-chrismc** [comment]
I don't feel like it's our place to set the release schedule

**Comment by mathbunnyru** [comment]
> I don't feel like it's our place to set the release schedule

We do not set it, merely ask to make a release almost 6 years after the last one :)

**Comment by prince-chrismc** [comment]
https://github.com/kazuho/picojson/pull/135#issuecomment-784240359

**Comment by mathbunnyru** [comment]
Thanks, @prince-chrismc

File: recipes/picojson/config.yml [hunk 1]
```diff
@@ -1,3 +1,5 @@
 versions:
+    "cci.20210117":
```
Comment (uilianries) [inline recipes/picojson/config.yml:2]: It would say to ask for a new release, but its development looks a bit slow.

**Review by uilianries — APPROVED** [review]

**Review by SSE4 — APPROVED** [review]

**Pull request merged**